src/Microsoft.DotNet.GitHub.IssueLabeler/appsettings.Development.json

```diff
@@ -5,19 +5,29 @@
       "threshold": "0.4",
       "prediction_url": "",
       "can_update_labels": false,
-      "can_comment_on": false
+      "can_comment_on": false,
+      "area_owners_doc": "https://github.com/dotnet/runtime/blob/main/docs/area-owners.md",
+      "new_api_pr_label": "new-api-needs-documentation",
+      "apply_linked_issue_area_label_to_pr": true,
+      "skip_prediction": false,
+      "skip_untriaged_label": true,
```

Comment:
Is this mean `untriaged` will not be added in runtime?

Reply:
Correct. We let fabricbot carry that responsibility.